### PR TITLE
feat: Add total summary page

### PR DESCRIPTION
**Pull request by malanxi2022**

### CODE REVIEW

**Comment by yihong0618** [comment]
also will be 2.5 feature
https://github.com/yihong0618/running_page/issues/720

**Comment by orionna319** [comment]
[total page 404](https://running-page-git-fork-malanxi2022-master-hongyis-projects.vercel.app/total), Maybe we should add the following configuration. [react-router-app-works-in-dev-but-not-after-vercel-deployment](
https://stackoverflow.com/questions/64815012/react-router-app-works-in-dev-but-not-after-vercel-deployment)
```json
{
  "rewrites":  [
    {"source": "/(.*)", "destination": "/"}
  ]
}
```
image

File: src/components/ActivityList/index.tsx [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+    // 计算 Y 轴的最大值和刻度
+    const yAxisMax = Math.ceil(Math.max(...data.map(d => parseFloat(d.距离))) + 10); // 取整并增加缓冲
+    const yAxisTicks = Array.from({ length: Math.ceil(yAxisMax / 5) + 1 }, (_, i) => i * 5); // 生成等差数列
+
+    return (
+        <div className={styles.activityCard}>
+            <h2 className={styles.activityName}>{period}</h2>
+            <div className={styles.activityDetails}>
+                <p><strong>总距离:</strong> {summary.totalDistance.toFixed(2)} km</p>
```
Comment (ben-29) [inline src/components/ActivityList/index.tsx:48]: We have a lot of international users, can you translate the text into English according to the [config](https://github.com/yihong0618/running_page/blob/fb98ab9c4edd69726a9bde34ffce458c66b7378e/src/utils/const.ts#L51)?

**Review by ben-29 — COMMENTED** [review]
It would be great if you could add an entry to the [navigation](https://github.com/yihong0618/running_page/blob/fb98ab9c4edd69726a9bde34ffce458c66b7378e/src/static/site-metadata.ts#L22)

**Pull request closed without merging**